permissions/resources.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line max-classes-per-file
+const User = class {
+  constructor(workerSid, roles) {
+    this.workerSid = workerSid;
+    this.roles = roles;
+  }
+};
+
+class Resource {
+  constructor(workerSid) {
+    this.workerSid = workerSid;
+  }
+}
+
+const ContactResource = class extends Resource {};
+const CaseResource = class extends Resource {};
```

Comment:
Why is this better than using a `Case`?  Just convenient naming of `workerSid`?

Reply:
I couldn't use `Case` in the beginning, but now I've managed to make it work.